src/main/java/io/github/cottonmc/cotton/gui/widget/WTextField.java

```diff
@@ -33,70 +33,90 @@
 import java.util.function.Predicate;
 
 public class WTextField extends WWidget {
-	public static final int OFFSET_X_TEXT = 4;
```

Comment:
Removing public fields/methods is technically api breaking, so could you readd this with `@Deprecated(forRemoval = true)`?